event_reporter/metrics/metrics.go

```diff
@@ -43,57 +45,54 @@ const (
 )
 
 var (
-	redisRequestCounter = prometheus.NewCounterVec(
-		prometheus.CounterOpts{
-			Name: "argocd_redis_request_total",
-			Help: "Number of kubernetes requests executed during application reconciliation.",
-		},
-		[]string{"initiator", "failed"},
-	)
-	redisRequestHistogram = prometheus.NewHistogramVec(
-		prometheus.HistogramOpts{
-			Name:    "argocd_redis_request_duration",
-			Help:    "Redis requests duration.",
-			Buckets: []float64{0.1, 0.25, .5, 1, 2},
-		},
-		[]string{"initiator"},
-	)
-	queueSizeCounter = prometheus.NewGaugeVec(
+	queueSizeGauge = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "cf_e_reporter_queue_size",
+			Name: "codefresh_event_reporter_queue_size",
 			Help: "Size of application events queue of taked shard.",
 		},
 		[]string{"reporter_shard"},
 	)
-	appEventsCounter = prometheus.NewCounterVec(
+
+	enqueuedEventsCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "cf_e_reporter_app_events_size",
-			Help: "Size of specific application events queue of taked shard.",
+			Name: "codefresh_event_reporter_enqueued_events_total",
+			Help: "Amount of specific application events queue of taken shard.",
 		},
-		[]string{"reporter_shard", "application", "got_in_queue", "error_in_learning_mode"},
+		[]string{"reporter_shard", "application", "error_in_learning_mode"},
 	)
-	erroredEventsCounter = prometheus.NewCounterVec(
+
+	droppedEventsCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "cf_e_reporter_errored_events",
-			Help: "Total amount of errored events.",
+			Name: "codefresh_event_reporter_dropped_events_total",
+			Help: "Amount of dropped application events queue of taken shard.",
```

Comment:
"Amount of application events not accepted into the queue of a particular shard."